gufe/tests/test_chemicalstate.py

```diff
@@ -0,0 +1,51 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import pytest
+from openff.toolkit.topology import Molecule
+
+import gufe
+
+
+@pytest.fixture
+def prot_comp(PDB_181L_path):
+    yield gufe.ProteinComponent.from_pdbfile(PDB_181L_path)
+
+
+@pytest.fixture
+def solv_comp():
+    yield gufe.SolventComponent(ions=('K', 'Cl'))
+
+
+@pytest.fixture
+def lig_comp(benzene_modifications):
+    yield gufe.SmallMoleculeComponent.from_rdkit(benzene_modifications['toluene'])
+
+
+def test_construction(prot_comp, solv_comp, lig_comp):
```

Comment:
Could you make this "test_complex_construction" and then add a test for at least the ligand in water case  (just so we have the very minimum of what we need to get everything working)

Reply:
done